internal/build/extend_build_test.go

```diff
@@ -0,0 +1,168 @@
+package build_test
+
+import (
+	"bytes"
+	"context"
+	"io"
+
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/apex/log"
+
+	"github.com/buildpacks/lifecycle/buildpack"
+
+	"github.com/buildpacks/pack/internal/build"
+	"github.com/buildpacks/pack/internal/build/fakes"
+
+	"github.com/docker/docker/api/types"
+	"github.com/golang/mock/gomock"
+	"github.com/heroku/color"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/buildpacks/pack/pkg/logging"
+	h "github.com/buildpacks/pack/testhelpers"
+
+	mockdocker "github.com/buildpacks/pack/internal/build/mockdocker"
```

Comment:
nit: can we group imports here?

Reply:
To be fixed in https://github.com/itsdarshankumar/pack/pull/1

Reply:
done!!